src/main/groovy/org/shipkit/internal/gradle/git/GitPullTask.java

```diff
@@ -0,0 +1,50 @@
+package org.shipkit.internal.gradle.git;
+
+import org.gradle.api.DefaultTask;
+import org.gradle.api.tasks.Input;
+import org.gradle.api.tasks.TaskAction;
+
+public class GitPullTask extends DefaultTask{
```

Comment:
Although we would keep the task internal for now (in internal package), let's add javadoc soon so that it's easier and faster to move it to public space when we are ready.